### DISCUSSION TITLE
Version 2.5.219 tarball does not install on Node 17

### DISCUSSION THREAD

**Discussion by jeswincredcore** [requarks/wiki#4721] [Error / Bug Report] [answered]
The tarball available for download (v2.5.219) at https://docs.requarks.io/install/linux does not install on Node 17.
The issue is an outdated version of extract-files (9.0.0) - related to subpath folder mapping. Newer versions seem to have this fixed. Please see link below [1].

Using subpath patterns in `node_modules/extract-files/package.json`  as follows fixes the issue:

```json
  "exports": {
    ".": {
      "import": "./public/index.mjs",
      "require": "./public/index.js"
    },
    "./public/index.js": "./public/index.js",
    "./public/extractFiles": "./public/extractFiles.js",
    "./public/isExtractableFile": "./public/isExtractableFile.js",
    "./public/ReactNativeFile": "./public/ReactNativeFile.js",
    "./package": "./package.json",
    "./package.json": "./package.json"
  }
```

[1]: I believe this is a related issue: https://github.com/jaydenseric/apollo-upload-client/issues/257#issuecomment-826535257

**Comment by NGPixel** [comment, marked as answer]
Odd-numbered Node.js versions are not supported. Please use Node.js 16.x instead.

**Reply by ericlighthofmann** [reply]
Hi @NGPixel  - This is also happening to me on a fresh install of v2.5.219 and Node 16.13.0. 

The above changes to the package.json fixes it for me.

**Comment by jeswincredcore** [comment]
Thanks @NGPixel. I'll switch to 16 then.
